firmware/application/freqman_db.hpp

```diff
@@ -56,6 +56,7 @@ enum class freqman_type : uint8_t {
     Single,    // f=
     Range,     // a=,b=
     HamRadio,  // r=,t=
+    Repeater,  // l=,t=
```

Comment:
Maybe I'm wrong, but I thought the existing HamRadio type with separate receive/transmit frequencies and tone was already meant for repeaters, except that the A/B frequencies are swapped on a Repeater versus on a Ham radio transceiver.  Every repeater I've used has also been activated by a CT tone, so it should filter out transmissions that are missing the tone (of course the tone is optional, but it should be supported at some point).  Do we really need a new freqman type for Repeaters?

Reply:
Secondly, if you do add a Repeater type, then I would propose that the record would indicate the Transmit frequency, Offset (signed number indicating difference between transmit & receive frequency [can be 0], and CT Tone.  This is the way Repeaters are usually defined.  For example, here's a list of repeaters in France:  https://www.repeaterbook.com/row_repeaters/Display_SS.php?state_id=FR&loc=%&call=%&use=%

Reply:
You might consider adding a call sign field too, or a call sign for the PortaPack in persistent memory.  Amateur radio repeaters transmit their callsign periodically per regulation (perhaps a future enhancement).  See examples in the table at the link above.

Reply:
HamRadio is designed so you can talk to it, while repeater is designed to relay signals.
While you and me are knowing the internals pretty well, we know the difference is thin. 
At the beginning I tried using HAMRADIO only, but it was getting confusing (for me ?).
So after a few tries I decided on adding a Repeater entry, that was more fitting the mind process behind it than using the same type for both modes.
I agree with the tone thing to open the channel. I was just not ready nor I know if I can do it nice on the PP X-D
Hence I left it like this. As it is, it's unusable for short signals anyway. I can only be used with repeated signals or long enough com starting by a silence/ a tone/ whatever is giving the PP enough time to switch to record mode and start recording.

> Secondly, if you do add a Repeater type, then I would propose that the record would indicate the Transmit frequency, Offset (signed number indicating difference between transmit & receive frequency [can be 0]), and CT Tone (optional). This is the way Repeaters are usually defined. For example, here's a list of repeaters in France: https://www.repeaterbook.com/row_repeaters/Display_SS.php?state_id=FR&loc=%&call=%&use=%

My bad ! I was thinking freq were uint64_t (dunno why). Off course I can store the offset instead ! The tone index should already be supported, even if not used yet.

> You might consider adding a call sign field too, or a call sign for the PortaPack in persistent memory. Amateur radio repeaters transmit their callsign periodically per regulation (perhaps a future enhancement). See examples in the table at the link above.
I'm all willing for it too, since it would allow people to use the PP as a half duplex relay in the norm.

I'm going to close the conversation. I don't think I can do the tone and callsign integration in that PR, but I'm already noting them for the next PR. I'll quickly open it as draft and I hope I'll find help for what I don't know yet in the regards of tones / callsigns.

I hope that with the incoming modification for the offset instead of a tx freq (I'm prepping  the commit on that PR now) you'll agree with me on the Repeater type :-)

Thanks for reading that heavy PR !